bullet-featherstone/src/SDFFeatures.cc

```diff
@@ -232,7 +232,7 @@ std::optional<Structure> ValidateModel(const ::sdf::Model &_sdfModel)
       for (std::size_t i = 0; i < model.LinkCount(); ++i)
       {
         const auto *link = model.LinkByIndex(i);
-        if (parentOf.count(link) == 0)
+        if (parentOf.size() != 0 && parentOf.count(link) == 0)
```

Comment:
I'm not sure why we would check the size here. `parentOf.size() == 0` implies that there is only one link and that link has no parent. In that case `parentOf.count(link) == 0` will be true. Inside of this branch we then double-check that if a `rootLink` was selected while iterating over the model then that root link is the same as the one we're currently looking at. I think it's reasonable to double-check that because if that's not true then there's an error in the logic of the parser.

Was there a specific situation that motivated these additional conditions? If so it would be helpful if I can reproduce it to figure out what might be wrong with the parser's logic.